src/Socket.cpp

```diff
@@ -30,7 +30,7 @@
 #ifdef HAVE_LIBSSL
 // Copypasted from
 // https://wiki.mozilla.org/Security/Server_Side_TLS#Intermediate_compatibility_.28default.29
```

Comment:
Is there some reason why we couldn't go to https://wiki.mozilla.org/Security/Server_Side_TLS#Modern_compatibility ? (Offtopic:) This was for both client & server, wasn't it?

I don't think anyone uses old OpenSSL/GnuTLS/whatever that wouldn't have those and IRCds which are actively developed has them and XChat support already dropped with SSLv3 support.